reana_client/openapi_connections/reana_server.json

```diff
@@ -1,182 +1,246 @@
 {
-  "definitions": {},
+  "definitions": {}, 
```

Comment:
@hjhsalo do you remember something related with trailing spaces when generating OpenAPI specs? it rings a bell that you solved it or you know how to solve it.

For some reason RS does it but RWC does not and the diff shows nothing. It is just a quick question if you don't remember straight away I will find out 🙃  

```diff
$ diff ~/reana/reana-server/scripts/generate_openapi_spec.py \
       ~/reana/reana-workflow-controller/scripts/generate_openapi_spec.py
38c38
< __title__ = "REANA Server"
---
> __title__ = "REANA Workflow Controller"
41c41
< __api_description__ = "Submit workflows to be run on REANA Cloud"
---
> __api_description__ = "Submit and manage workflows"
```